### ISSUE TITLE
### <span aria-hidden="true">❌</span> Deploy Preview for *cfccc* failed.

### ISSUE THREAD

**Issue by t9611558-star** [t9611558-star/CFC#7]
### <span aria-hidden="true">❌</span> Deploy Preview for *cfccc* failed.

|  Name | Link |
|:-:|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | 63a7c53c1cf5eff252462b2200ce5f0db01f5c76 |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/projects/cfccc/deploys/6aa5683998e7210008ea452a |

_Originally posted by @netlify[bot] in https://github.com/t9611558-star/CFC/issues/5#issuecomment-5646650147_
